py/server/deephaven/table.py

```diff
@@ -985,7 +989,7 @@ def where_one_of(self, filters: Union[str, Sequence[str]] = None) -> Table:
             filters = to_sequence(filters)
             with _query_scope_ctx():
                 return Table(
-                    j_table=self.j_table.where(_JFilterOr.of(_JFilter.from_(*filters)))
+                    j_table=self.j_table.where(j_array_list([_JFilterOr.of(_JFilter.from_(*filters))]))
```

Comment:
You point out that with the current set of overloads you don't need the `j_array_list`.